messages/JsonMessages/CompanionTextures.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+import { extendApi } from "@anatine/zod-openapi";
+
+/*
+ * WARNING! The original file is in /messages/JsonMessages.
+ * All other files are automatically copied from this file on container startup / build
+ */
+
+//The list of all the companion textures
+
+export const companionTexture = z.object({
+    name: extendApi(z.string(), { description: "The name of the texture.", example: "dog1" }),
+    img: extendApi(z.string(), {
+        description: "The URL of the image of the texture.",
+        example: "http://example.com/resources/characters/pipoya/Cat 01-1.png",
+    }),
+    behaviour: extendApi(z.enum(["simple", "dog", "cat"]), {
+        description: "The companion behavior (simple|dog|cat)",
+        example: "dog",
+    }),
+});
```

Comment:
Hey @taksan,

I have several questions/remarks on this.

The "behaviour" parameter seems interesting but is currently unused. Do you plan anything special afterwards? (if yes, I'm curious to know what ;), if no, we can probably remove this)

Also, as you probably noticed, we refactored the "select your woka" and "customize your woka" scenes, but we did not refactor the "companion" scene. We will do this in the future, obviously.

Right now, the list of wokas looks something like this:

```json
[
      {
        "name": "Default Collection",
        "textures": [
          {
            "id": "94f39337-4da3-47b5-b8e1-b9528aa2698a",
            "name": "Accessory 1",
            "url": "https://play.staging.workadventu.re/resources/customisation/character_accessories/character_accessories1.png"
          },
          {
            "id": "8453d808-2e39-48a4-abdb-c91b0bf2b4e0",
            "name": "Accessory 2",
            "url": "https://play.staging.workadventu.re/resources/customisation/character_accessories/character_accessories2.png"
          },
          {
            "id": "ba860312-0dfc-4786-b911-499cf4d0b4eb",
            "name": "Accessory 23",
            "url": "https://play.staging.workadventu.re/resources/customisation/character_accessories/character_accessories23.png"
          },
        ]
      }
    ]
```

In the new Woka scene, Woka can belong to several "collections". We will probably use the same "collections" mechanism for companions (we could categorize companions as "Cats", "Dogs", "Robots", "Pokemons" or even "NFTs" for instance.

So even if the companions list does not **yet** support the notion of categories, it might be a good idea to add those right now. This will avoid having to change the Admin API signature in a future release.